src/components/Content/Content.stories.tsx

```diff
@@ -129,32 +154,26 @@ export const Playground = (): ReactElement => {
 		},
 	];
 
-	/**
-	 * Simulates a DB calls that fetch the values for
-	 * each field.
-	 * @returns the resolved values
-	 */
-	const getValues = async (): Promise<MosaicObject> => {
-		return new Promise((resolve) => {
-			setTimeout(() => {
-				if (useValues) {
-					resolve(values);
-				} else {
-					resolve(undefined);
-				}
-			}, 500);
-		});
-	};
+	let oneColumToDisplay;
+	let twoColumnsToDisplay;
+	let buttonsToDisplay;
+
+	if (fields && data) {
```

Comment:
Fields and data are not being changed and always exist so this will always be true. I believe we could change this to be moved to a function that gets called in line 174 (in sections = {}) that way we can also reduce the amount of code in that single line to make it easier for others to read / understand.